### DISCUSSION TITLE
Missing listing id in orders

### DISCUSSION THREAD

**Discussion by Faaque** [etsy/open-api#1282] [Q&A]
I would like to highlight that the listingId is absent from the sales receipt for certain orders. We are utilizing the Receipt API to download sales orders on behalf of our client. Could you please check why the ListingID is missing for these orders?

3083827729
3083831843

**Comment by amitkumar-dev92** [comment]
Hello @Faaque 
Hope you are using the [getShopReceipt](https://developers.etsy.com/documentation/reference#operation/getShopReceipt) API to check the order details. 
this call can only be made for shared receipt_ids when anyone having access_token and shop_id. 
Till now I haven't faced any such issue with any of my clients.
you could mail to Etsy dev team, certainly they can help you on it in better way
thanks
